### ISSUE TITLE
Both go 1.9 and 1.8.3 are downloaded

### ISSUE THREAD

**Issue by ivyxyz** [bazelbuild/rules_go#771]
Go 1.9 is the default version now, but `go_repository_tools()` specifies `go1_8_3_linux_amd64` and downloads Go 1.8.3. Can go_repository_tools follow DEFAULT_VERSION in toolchains.bzl or accept a go_version argument?

https://github.com/bazelbuild/rules_go/blob/master/go/private/repository_tools.bzl#L34

**Comment by ianthehat** [comment]
Just to let you know, I have [a fix](https://github.com/ianthehat/rules_go/tree/repository_tools_sdk) for this already done, but it's stuck behind some other changes that need to land first.
